catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/TreUtility.java

```diff
@@ -73,4 +79,42 @@ public static Integer convertToInteger(Tre tre, String fieldName) {
         }
         return Optional.empty();
     }
+
+    public static Float convertToFloat(Tre tre, String fieldName) {
+        Serializable value = TreUtility.getTreValue(tre, fieldName);
+        if (value instanceof String) {
+            return Float.valueOf((String) value);
+        } else {
```

Comment:
Will this always be true? If so, why is `TreUtility.getTreValue()` not returning `String` instead of `Serializable`?